device/cuda/src/sanity/contiguous_on.cuh

```diff
@@ -60,8 +61,8 @@ template <std::equality_comparable T>
 __global__ void is_contiguous_on_all_unique(
     vecmem::data::vector_view<T> in_view, bool* out) {
 
-    int tid_x = threadIdx.x + blockIdx.x * blockDim.x;
-    int tid_y = threadIdx.y + blockIdx.y * blockDim.y;
+    const unsigned int tid_x = threadIdx.x + blockIdx.x * blockDim.x;
+    const unsigned int tid_y = threadIdx.y + blockIdx.y * blockDim.y;
```

Comment:
These should be `global_index_t`.

Reply:
I was thinking about it... 🤔 I'm happy to turn these into `global_index_t` if you think that's the correct way about this, but in the end these are not really "global" indices here.

Still, I have no strong opinion about this. 🤔

Reply:
In what way are they not global indices? Or do you mean that the global index is strictly a _linear_ index in the global thread space? And because these are 2D they do not meet that description?